home/css/layout.scss

```diff
@@ -278,6 +278,12 @@ footer {
         margin-right: 8px;
     }
 
+    .bluesky-icon {
```

Comment:
It would be nice to use font awesome to display the icon instead of svg.

Reply:
Updated to use the fontawesome icon